jobs/testable.rb

```diff
@@ -0,0 +1,27 @@
+require 'dashing'
+require File.expand_path('../../lib/travis_backend', __FILE__)
+
+SCHEDULER.every '1d', first_in: '1s' do |_job|
+  travis_backend = TravisBackend.new
+  github_backend = GithubBackend.new
+
+  testable_repos = github_backend.testable_repos
+
+  travis_repos = testable_repos.map { |repo| travis_backend.get_repo(repo) }
+  repos_without_builds = travis_repos.select { |repo| repo['last_build_id'].nil? }
+  ignored_repos = ENV['TRAVIS_IGNORED_REPOS'].split(',').compact
+
+  items = repos_without_builds.map do |repo|
```

Comment:
Yeah, the original repo that I forked is not in the best shape in terms of code or tests. I'm planning on addressing that in subsequent PRs. Not sure if you read my commit message, but it explains what I'm doing here.

I should rename the file to `testable_without_ci.rb` to make it clear that this widget only contains repos that are testable, but don't have builds running in a CI server.

To summarize what I did:
Before, the Travis widget (using the original code that I forked) was displaying all repos belonging to 18F, whether or not they actually had builds running on Travis. What I did here was modify that widget to ignore repos that don't have builds on Travis.

Then, to be able to see which repos actually *should* have builds running on Travis, but don't, I created a new widget that determines those repos based on the `testable` status from the team-api, and based on whether or not they have builds running on Travis.

Does that make sense?